### DISCUSSION TITLE
Error when using multiple Rule on one property

### DISCUSSION THREAD

**Discussion by ductho1201** [livewire/livewire#6300] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.7

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Define multiple Rule in one Property:
    #[Rule('required', message: 'Password cannot empty')]
    #[Rule('min:6', message: 'Password must have 6 and more character')]
    public string $password = "";
2. In form, use wire:model:
    <input type="password" wire:model="password" class="form-control" />
    @error('password')
      <div class="red invalid-feedback">{{ $message }}</div>
    @enderror
3. Don't put any value to password input field then submit. The validate function work fine.
    public function handleLogin(): void
    {
        $validated = $this->validate();
        $user = app(IAuthService::class)->handleLogin($validated);
        if (!$user) {
            $this->successMessage = null;
            $this->errorMessage = 'Credentials are incorrect ';
            return;
        }
        $this->errorMessage = null;
        $this->successMessage = 'Login success';
    }
4. Put one character in password input field, got error: 
Undefined array key "string".
The function make error in: vendor/laravel/framework/src/Illuminate/Validation/Concerns/FormatsMessages.php
    protected function getInlineMessage($attribute, $rule)
    {
        $inlineEntry = $this->getFromLocalArray($attribute, Str::snake($rule));
        return is_array($inlineEntry) && in_array($rule, $this->sizeRules)
                    ? $inlineEntry[$this->getAttributeType($attribute)]
                    : $inlineEntry;
    }

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by citricguy** [comment]
Have you tried to composer update? This is working for me in the most recent beta now.
